README.md

```diff
@@ -111,7 +111,10 @@ spec:                  # spec (used to be che-plugin.yaml)
   containers:          # optional; sidecar containers for plugin
     - image:
       name:              # name used for sidecar container
-      memorylimit:       # Kubernetes/OpenShift-spec memory limit string (e.g. "512Mi")
+      memoryLimit:       # Kubernetes/OpenShift-spec memory limit string (e.g. "512Mi")
```

Comment:
Consider adding a reference to k8s docs https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory